src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,77 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import IconCaretRight from "@elements/Icon/Generated/IconCaretRight";
+import IconCheck from "@elements/Icon/Generated/IconCheck";
+import { IconSize } from "@elements/Icon/IconSize";
+import { merge } from "@utilities/merge";
+import React, { FC } from "react";
+import { MenuItemContent, MenuItemContentProps, MenuItemContentSize } from "./MenuItemContent";
+
+export enum MenuItemStyle {
+    Primary = "Primary",
+    Danger = "Danger",
+}
+
+export enum SelectionIndicatorIcon {
+    Check = "Check",
+    CaretRight = "CaretRight",
+}
+
+export type MenuItemProps = {
+    style?: MenuItemStyle;
+    size?: MenuItemContentSize;
+    disabled?: boolean;
+    active?: boolean;
+    selectionIndicator?: SelectionIndicatorIcon;
+} & Omit<MenuItemContentProps, "iconSize">;
+
+export const getStyle = (isDangerStyle: boolean, disabled: boolean, active: boolean): string => {
```

Comment:
could also be done with inline checks:
```
export const getStyle = (isDangerStyle: boolean, disabled: boolean, active: boolean): string =>
    merge([
        disabled && `bg-black-0 pointer-events-none ${isDangerStyle ? "text-red-40" : "text-black-40"}`,
        active && `font-medium ${isDangerStyle ? "text-red-70" : "text-black"}`,
        !disabled && !active && (isDangerStyle ? "text-red-60" : "text-black-80"),
    ]);
```

Reply:
and maybe rename to `getColorStyles`, since it only affects the color and the rest of the styles are done inline. wdyt?